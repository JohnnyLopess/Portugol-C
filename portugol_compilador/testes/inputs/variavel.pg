programa
{
inicio
var: real numero
var: caractere letra
var: inteiro idade
escreva("Digite um nÃºmero real: ")
leia(numero)
escreva("Digite uma letra: ")
leia(letra)
escreva("Digite sua idade: ")
leia(idade)
escreva(numero)
escreva(letra)
escreva(idade)
fim
}