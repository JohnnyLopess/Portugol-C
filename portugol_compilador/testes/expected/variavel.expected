#include <stdio.h>
#include <string.h>

int main() {
    float numero;
    char letra;
    int idade;
    printf("Digite um n√∫mero real: ");
    scanf("%f", &numero);
    printf("Digite uma letra: ");
    scanf(" %c", &letra);
    printf("Digite sua idade: ");
    scanf("%d", &idade);
    printf("%f\n", numero);
    printf("%c\n", letra);
    printf("%d\n", idade);
    return 0;
}
